alf/environments/suite_mario.py

```diff
@@ -30,7 +30,13 @@
 
 
 def is_available():
-    return retro is not None
+    if retro is None:
+        return False
+    try:
+        retro.data.get_romfile_path('SuperMarioBros-Nes')
+    except:
```

Comment:
In general, we should not use "except:" because it catches all kinds of error include syntax errors. Should use an explicit Exception type.